.github/scripts/generate_ci_workflows.py

```diff
@@ -576,6 +595,16 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels=set([LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CPU]),
         ),
     ),
+    CIWorkflow(
+        arch="linux",
+        build_environment="linux-bionic-rocm4.5-py3.7",
+        docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-bionic-rocm4.5-py3.7",
+        test_runner_type=LINUX_ROCM_TEST_RUNNER,
+        num_test_shards=2,
+        ciflow_config=CIFlowConfig(
+            labels=set([LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CUDA]),
```

Comment:
```suggestion
            labels=set([LABEL_CIFLOW_LINUX, LABEL_CIFLOW_ROCM]),
```

Let's add another CIFLOW label above and then limit these only to run when `LABEL_CIFLOW_ROCM` is actually active

Reply:
59dbc9c2108331dd02ee845371c67a58f61d962c creates the new LABEL_CIFLOW_ROCM.